tox.ini

```diff
@@ -65,6 +66,22 @@ deps =
     types-jmespath
 commands = mypy src/
 
+[testenv:mypy_untyped_defs]
+setenv =
+    # When this number decreases to 0,
+    # update mypy.ini to disallow untyped defs completely,
+    # then remove this test environment.
+    MAX_UNTYPED_DEFS_FILE_COUNT = 148
+allowlist_externals =
+    bash
+deps =
+    mypy==0.991
+    types-jwt
+    types-requests
+    types-jmespath
+commands =
+    bash -ec 'if [ "$(mypy --disallow-untyped-defs src/ | grep -oE "Found.+([0-9]+) errors.+([0-9]+) files" | cut -d\  -f5)" -gt $MAX_UNTYPED_DEFS_FILE_COUNT ]; then echo -e "\n\nThe number of files with untyped definitions has increased.\n\n"; false; fi'
+
```

Comment:
As you mentioned yesterday, this is a viable workaround for quickly and easily getting this working.
I'm fine with it, in part because I'd like to see us make strides towards removing it (so the discomfort we might feel with "bash in tox" is offset by the fact that it is meant to have a limited lifetime).

Even so, I'm not sure what I can do other than laugh!
Good thing we don't have a long-lines linter for our config files! 😂

Reply:
Hmm, this check no longer works after rebasing on `main`. I'll take a quick look at this.